### PR TITLE
Swift 2.2 + refactor ViewCells and sample controller

### PR DESCRIPTION
**Pull request by kdazzle**
Sorry, this is kind of a large and unwieldy PR, but I merged the DateViewCell and ContentViewCell into one class, which allowed for some duplication with cell set up to be removed. 

Also made a wrapper for NSIndexPath because this library turns sections into rows and rows into columns, so the property names were really confusing. There's probably a better way of doing that, but this is at least nicer than it was.

And I made some adjustments for Swift 2.2 + broke a large method in the sample controller down into multiple methods.

### CODE REVIEW

File: CustomCollectionLayout/CustomCollectionViewLayout.swift [hunk 1]
```diff
@@ -1,19 +1,11 @@
-//
-//  CustomCollectionViewLayout.swift
-//  CustomCollectionLayout
-//
-//  Created by JOSE MARTINEZ on 15/12/2014.
-//  Copyright (c) 2014 brightec. All rights reserved.
-//
```
Comment (kdazzle) [inline CustomCollectionLayout/CustomCollectionViewLayout.swift]: I realized too late that I deleted this chunk. It was a reflex, I'm sorry.

File: CustomCollectionLayout/MultiColumnIndexPath.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+
+/**
+ Wrapper around NSIndexPath because NSIndexPath.sections become rows under
+ multiple columns, and NSIndexPath.rows are effectively columns.
+ */
+public class MultiColumnIndexPath {
```
Comment (kdazzle) [inline CustomCollectionLayout/MultiColumnIndexPath.swift]: This is the wrapper class

**Comment by farazhaider88** [comment]
how can i get this code?

**Comment by kdazzle** [comment]
@farazhaider88 checkout the branch at https://github.com/kdazzle/CustomCollectionViewLayout

**Pull request closed without merging**